notebooks/18_deep_neural_network/index.md

```diff
@@ -0,0 +1,201 @@
+# Deep Neural Networks
```

Comment:
Try to have a different format for title and write authors' names in your lecture note

Reply:
The authors' names are not written yet!